web/l10n/en.json

```diff
@@ -425,5 +425,10 @@
     "categoryTruck": "Truck",
     "categoryVan": "Van",
     "maintenanceStart": "Start",
-    "maintenancePeriod": "Period"
+    "maintenancePeriod": "Period",
+    "twofa": "2FA",
```

Comment:
Sting names are all over the place. Please us "login" prefix.

Reply:
Fixed this. Prefixed those used in the user dialog with 'user' and those used in the login dialog with 'login'.